src/IECoreHoudini/GEO_CortexPrimitive.cpp

```diff
@@ -387,26 +376,77 @@ GA_PrimitiveTypeId GEO_CortexPrimitive::typeId()
 	return m_definition->getId();
 }
 
-#if UT_MAJOR_VERSION_INT >= 14
+#if UT_MAJOR_VERSION_INT >= 16
+
+void GEO_CortexPrimitive::createCortexPrimBlock(
```

Comment:
This seems like new behaviour. Is this required for H16 or optional for a performance gain? If its the later, it should be split out into its own commit.